etc/proxy-server.conf-sample

```diff
@@ -75,9 +75,15 @@ use = egg:swift3#swift3
 # This stuff is still under development and it might cause something
 # you don't expect.
 # s3_acl = false
+#
 # If set to 'true', Swift3 will append Swift3 ACL from Bucket to Object ACL
 # when an object is stored. Requires s3_acl.
 # s3_acl_inherit = false
+#
+# If set to 'true', Swift3 won't check ACLs before executing any request.
+# DO NOT ENABLE THIS ON A PUBLIC-FACING GATEWAY. It is intended only
```

Comment:
This will effectively give any authenticated user a read/write access to all buckets and objects of its tenant/project/account.

Do not activate it unless you have an additional layer that handles ACLs on top!